db/seeds/questions.yml

```diff
@@ -109,6 +120,27 @@ design_portfolio:
 
 # Short answer questions
 
+fa20_mission_first:
+  title: 'What does social good mean to you?'
+  hint: "If you're applying for a Developer/Designer position, please include: Why do you want to build/design applications in the social impact space?<br/><br/>If you're applying for an External position, please include: Why do you want to promote tech for social good?<br/><br/>Do not use this space to talk about Blueprint — we want to hear about you! Please limit your response to 200 words."
```

Comment:
Change Developer/Designer to Developer or Designer.

Not a huge deal but I could see this being misinterpreted as roles can be blended